experimental/Schemes/BlowupMorphism.jl

```diff
@@ -96,108 +97,191 @@ function strict_transform(p::BlowupMorphism, inc::CoveredClosedEmbedding)
   return Z_trans, inc_Z_trans, pr_res
 end
 
+#@attr Bool function is_one(I::Union{MPolyQuoIdeal, MPolyLocalizedIdeal, MPolyQuoLocalizedIdeal})
+#  return is_one(saturated_ideal(I))
+#end
+
+@doc """
+    strict_transform(p::BlowupMorphism, I::IdealSheaf)
+
+For a `BlowupMorphism`  ``p : Y → X`` and an `IdealSheaf` ``I`` on ``X`` return the
+strict transform of ``I`` on ``Y``.
+"""
 function strict_transform(p::BlowupMorphism, I::IdealSheaf)
+  Istrict,_ =_do_transform(p,I,-1)
+  return Istrict
+end
+
+@doc """
+    weak_transform(p::BlowupMorphism, I::IdealSheaf)
+
+For a `BlowupMorphism`  ``p : Y → X`` and an `IdealSheaf` ``I`` on ``X`` return the
+weak transform ``J`` of ``I`` on ``Y``, i.e. an `IdealSheaf` satisfying ``E^m J = I`` with ``m``
+maximal and ``E`` the 'IdealSheaf' of the exceptional divisor of ``p``.
```

Comment:
$J$ and $E$ are ideal sheaves on $Y$ while $I$ is a sheaf of ideals on $X$. So what exactly do you mean by
$E^m J = I$?

Reply:
Typo corrected: p^*I